components/SortMenu.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import { Menu, Button, Typography, Icon } from "@equinor/eds-core-react";
+import { chevron_down } from "@equinor/eds-icons";
+
+export default function SortMenu(props: {
```

Comment:
I would probably just use an `<option>`-tag instead. Ref https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option
But this works fine. :+1: